### DISCUSSION TITLE
Is there a way to disable extensions for the sole purpose of running Lighthouse tests from inside of DevTools?

### DISCUSSION THREAD

**Discussion by rachelslurs** [GoogleChrome/lighthouse#16109] [Q&A] [answered]
I recently noticed some of my extensions showing up in my Lighthouse report as being unminified. Is there a setting I can use to ignore these when running in my Chrome DevTools?

![Screenshot 2024-07-11 at 11 55 23 AM](https://github.com/GoogleChrome/lighthouse/assets/1870406/3107c3c2-8a7f-44c6-8409-ec7d185382a1)

**Comment by connorjclark** [comment, marked as answer]
The only option available today is to prefer incognito (or an otherwise fresh chrome profile).

https://github.com/GoogleChrome/lighthouse/issues/4878#issuecomment-2143270637

Lighthouse doesn't exclude them from table results, but we can't truly exclude their impact and our policy has been to therefore not hide them.

We consider them as third-party in the "third party results" toggleable filter, but that [only shows](https://github.com/GoogleChrome/lighthouse/blob/a21fde7f0e997802d6bd574edf156bb04649ecf9/report/renderer/report-ui-features.js#L285-L288) when there are 3p AND non-3p results in an audit (so in your example it doesn't display...)

We do emit this warning when we can detect that a lot of time was spent in extension code:

> Chrome extensions negatively affected this page's load performance. Try auditing the page in incognito mode or from a Chrome profile without extensions.
